src/routing/dragon.ts

```diff
@@ -3,6 +3,7 @@ import PrefixMap from './prefix-map'
 import { create as createLogger } from '../common/log'
 const log = createLogger('dragon')
 import { Relation, getRelationPriority } from './relation'
+import ForwardingRoutingTable from '../services/forwarding-routing-table';
```

Comment:
Linting fails due to semicolon. Also, the `PrefixMap` import on line 2 isn't needed anymore.